pages/recordbag.js

```diff
@@ -14,7 +14,7 @@ export default function RecordBag({ selection, onHandleChange }) {
             <RecordFile
               key={file.CatalogId}
               record={file}
-              onHandleChange={onHandleChange}
+              onChange={onChange}
```

Comment:
```suggestion
              onToggleBookmark={onToggleBookmark}
```

Reply:
Thank you @NikRoe – I made the suggested change and renamed the onChange function in all files. Unfortunately I now get a runtime errror that onToggleBookmark is not a function. I double checked for typos and if the passing of the function is applied correctly everywhere. I also tried to destructure onToggleBookmark when being called within the onChange attribute in the component but that didn't help either. Also googling didn't help. Would you have an idea what might be the issue here? 
Many thanks in advance!

<img width="506" alt="Bildschirmfoto 2022-10-04 um 11 17 47" src="https://user-images.githubusercontent.com/104068626/193784084-a3315ddf-7ade-445a-992d-06712caf0e53.png">

Reply:
I think this issue comes from line 17 in your `recordbag.js` file.
There it says: 
```js
onChange={onToggleBookmark}
```

but it should be 
```js
onToggleBookmark={onToggleBookmark}
```

Reply:
Ahhh super, thank you! I understand what the error was now. I had thought of the recordbag as a component when it is just a page that passes through the function as props. Works now!